Source/Classes/View/CardBalance/CardBalance.swift

```diff
@@ -81,8 +81,16 @@ public class CardBalance: UIView {
             return
         }
         
-        balanceTitle.font = UIFont.systemFont(ofSize: titleFontSize, weight: .regular)
-        balanceTitle.text = model.title.message
+        if model.title != nil {
+            guard let title = model.title else { return }
+            balanceTitle.font = UIFont.systemFont(ofSize: titleFontSize, weight: .regular)
+            balanceTitle.text = title.message
+            setupLabelColors(balanceTitle, field: title)
+        } else {
+            balanceTitle.text = ""
+        }
```

Comment:
I think checking `!= nil` is a bit boilerplatey, and the `else` cause shouldn't be needed (if we don't set the labels text, it should be empty by default). 

What do you think of changing this to an `if let` clause, something like: 

```
if let title = model.title {
    balanceTitle.text = title
    (...)
}
```

Reply:
Thanks for the help !
I didn't think about that